orchestrator/gorc/keystore.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN PRIVATE KEY-----
+MIGEAgEAMBAGByqGSM49AgEGBSuBBAAKBG0wawIBAQQgbnlpnqTEgc+m76CKUu5o
+vg70vadRVsZfBQQ/KS+7SOShRANCAAS5t4YZqM+ndF+atVjOvD8VRlt/iSDbbIq7
+boxLzSxnruYXrvlbqC4VDToBAfWc+At6rdlBGGDcq7Mvz7FjlPxF
+-----END PRIVATE KEY-----
```

Comment:
I think we should `git rm` this and add `*.pem` to our project's .gitignore to ensure we don't expose "real" private keys.

Reply:
Ok, I'll do that